contrib/gofiber/fiber.v2/fiber.go

```diff
@@ -0,0 +1,70 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// Package fiber provides tracing functions for tracing the fiber package (https://github.com/gofiber/fiber).
+package fiber // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/gofiber/fiber.v2"
+
+import (
+	"fmt"
+	"math"
+	"net/http"
+	"strconv"
+
+	"github.com/gofiber/fiber/v2"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// Middleware returns middleware that will trace incoming requests.
+func Middleware(opts ...Option) func(c *fiber.Ctx) error {
+	cfg := new(config)
+	defaults(cfg)
+	for _, fn := range opts {
+		fn(cfg)
+	}
+	return func(c *fiber.Ctx) error {
+		opts := []ddtrace.StartSpanOption{
+			tracer.SpanType(ext.SpanTypeWeb),
+			tracer.ServiceName(cfg.serviceName),
+			tracer.Tag(ext.HTTPMethod, c.Method()),
+			tracer.Tag(ext.HTTPURL, c.Request().URI().String()),
+			tracer.Measured(),
+		}
+		if !math.IsNaN(cfg.analyticsRate) {
+			opts = append(opts, tracer.Tag(ext.EventSampleRate, cfg.analyticsRate))
+		}
+
+		opts = append(opts, cfg.spanOpts...)
+		span, _ := tracer.StartSpanFromContext(c.Context(), "http.request", opts...)
+		defer span.Finish()
+
+		resourceName := c.Path()
+		if resourceName == "" {
+			resourceName = "unknown"
+		}
+		resourceName = c.Method() + " " + resourceName
+		span.SetTag(ext.ResourceName, resourceName)
+
+		//pass the execution down the line
```

Comment:
```suggestion
		// pass the execution down the line
```
Please do this everywhere on comments. A space is needed after `//`

Reply:
done